org.eclipse.winery.yaml.converter/src/test/java/org/eclipse/winery/yaml/converter/xml/Showcase.java

```diff
@@ -43,7 +43,7 @@ public void zipTypeTest() throws Exception {
         InputStream fis = new FileInputStream(path);
         IRepository repository = RepositoryFactory.getRepository(Utils.getTmpDir(Paths.get("repository")));
         CsarImporter csarImporter = new CsarImporter();
-        csarImporter.readCSAR(fis, true, true);
```

Comment:
Again here, we plan to use a new class to encapsulate the set of options that might be applicable while importing the csar (CsarImportOptions). For now this class includes the following attributes:
```
boolean addToProvenance;
boolean asyncWPDParsing;
boolean validate;
```
But when @v-yussupov integrates the security stuff, the options can also go here. The class is intended to capture all options that can be enforced externally (passed to the rest api as query params).
Notice that having multiple methods instead of the 3 existing boolean parameters means 2^3=8 method (to capture all possible combinations). Furthermore, by having this single object that represent the options we 'fixed' the signature of multiple other methods that depended on this one and thus also had multiple boolean parameters (now all of them have the CsarImportOptions as a parameter)